src/merge.js

```diff
@@ -243,8 +243,9 @@ class Merge {
         doc.creationDate = file.creationDate
         if (this.sameBinary(file, doc)) {
           if (doc.size == null) { doc.size = file.size }
-          if (doc.class == null) { doc.class = file.class }
-          if (doc.mime == null) { doc.mime = file.mime }
+
+          doc.class = file.class
+          doc.mime = file.mime
```

Comment:
Actually I may be wrong too.
Better see this change's context, including [this line](https://github.com/cozy-labs/cozy-desktop/pull/555/files/b9139c4d29bb61e5fd859563aba789e3ccd1901f#diff-d33c8624f2fb2ac1fc83292e027196dbL241)